src/axom/spin/internal/linear_bvh/math.hpp

```diff
@@ -0,0 +1,224 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#ifndef AXOM_SPIN_BVH_MATH_H_
+#define AXOM_SPIN_BVH_MATH_H_
+
+#include "axom/core/Macros.hpp"
+#include "axom/core/Types.hpp"    // for fixed bitwidth types
+
+// include math so we can use functions defined
+// in both cuda and c
+#include <math.h>
+
+#define DRAY_INF_32 0x7f800000U
```

Comment:
You can probably change this from dray (devil ray) to something more axom specific

Reply:
Good catch! It was one of the clean up items I intended to do that escaped my radar. I change the prefix to `AXOM_` instead and made them `constexpr` variables that are guarded within the namespace. They are only used within this file.